tests/regress/jumping.py

```diff
@@ -68,20 +66,20 @@ def hook_block(self, uc, address, size, _):
     # callback for tracing instructions
     def hook_code(self, uc, address, size, _):
         insn = uc.mem_read(address, size)
-        regress.logger.debug(">>> Tracing instruction at %#x : %s", address, binascii.hexlify(insn))
+        regress.logger.debug(">>> Tracing instruction at %#x : %s", address, insn.hex())
 
         regs = uc.reg_read_batch((
             UC_X86_REG_RAX, UC_X86_REG_RBX, UC_X86_REG_RCX, UC_X86_REG_RDX,
             UC_X86_REG_RSI, UC_X86_REG_RDI, UC_X86_REG_RBP, UC_X86_REG_RSP,
-            UC_X86_REG_R8,  UC_X86_REG_R9,  UC_X86_REG_R10, UC_X86_REG_R11,
+            UC_X86_REG_R8, UC_X86_REG_R9, UC_X86_REG_R10, UC_X86_REG_R11,
```

Comment:
Yet another example for a case where blindly sticking to strict styling rules doesn't make things more readable.